module/connection/ServerDialog.kt

```diff
@@ -113,7 +119,7 @@ object ServerDialog {
                     UsernameFormField(state)
                     PasswordFormField(state)
                     TLSEnabledFormField(state)
-                    if (state.tlsEnabled) CACertificateFormField(state)
+                    if (state.tlsEnabled) CACertificateFormField(state = state, dialogWindow = window)
```

Comment:
It seems ever so slightly awkward that we pass in `window` here but not to the other form fields. Somehow, it feels like it should be part of the context of `ServerDialog` (ie a class field) rather than being a parameter. But `ServerDialog` is an object, not a class; and so are all the other `Dialog`s in our modules. In conclusion, we won't take any action now.